src/System.Net.NetworkInformation/tests/FunctionalTests/PhysicalAddressTest.cs

```diff
@@ -73,7 +73,7 @@ public void PhysicalAddress_Clone_Pass()
         }
 
         [Theory]
```

Comment:
In the test(s) that check for this FormatException, you could do string.Contains to confirm it's within the message somewhere.